More updates to package

shgysk8zer0 opened the pull request:
## Update package info
### List of significant changes made
- Update homepage
- Update min NPM version
- Rename `update` script to `update-packages`
- Add `update-remote` script to pull in lastest versions of packages
- Re-enable `lint:js` test

shgysk8zer0 committed:
More updates to package

- Update homepage
- Update min NPM version
- Rename `update` script to `update-packages`
- Add `update-remote` script to pull in lastest versions of packages
- Re-enable `lint:js` test

kvsun-editor reviewed (approved):
If this passes tests, that'll be great. Been wanting to lint some JS for a while now.

shgysk8zer0 committed:
Update NPM version in .travis.yml

shgysk8zer0 committed:
Set server distro

Would like to use something more recent than 14.04, but Trusty is
what is listed as being available.

shgysk8zer0 commented:
Finally! Travis-CI really needs to do something about all of their out-dated software.
